### ISSUE TITLE
Enhance wins JavaScript logic for assigning alt text to badge icons 

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#3960]
### Overview

To comply with Web Content Accessibility Guidelines (WCAG), we need to ensure that the alt text for the wins badge icons is meaningful and is displayed only when appropriate. On the intial page wins page load, JavaScript must be modified to display meaningful alt text, and on the modal overlay ("See More"), Javascript must be modified to remove the alt text.

### Action Items

- [x] Identify the 2 pertinent lines of JavaScript by searching for "alt" in the file https://github.com/hackforla/website/blob/gh-pages/assets/js/wins.js
- [x] Note that both lines of code are used to generate html both the initial wins page (div.wins-item-container) and the overlay modal (div.overlay-item-container), based on the value of the variable 'view'.  
- [x] For view=wins, change the JavaScript so the alt text is identical to the tooltip text
- [x] For view=overlay, change the JavaScript to remove the alt text (ie set alt="")
- [x] In the `toggleSeeMoreLess` function, add code to remove/restore the alt text as appropriate, as the mobile/tablet layout changes in response to the user clicking"See More/See Less".

### Resources/Instructions

wins JavaScript code:  https://github.com/hackforla/website/blob/gh-pages/assets/js/wins.js
For background on this issue, see https://github.com/hackforla/website/issues/2569
[Wins page](https://www.hackforla.org/wins/)
[WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
[FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

**Comment by ExperimentsInHonesty** [comment]
@roslynwythe please add this project to the project board

**Comment by RyanGehris** [comment]
Availability: Monday-Friday 5-9pm
ETA: 02/28/2023 EOD

**Comment by RyanGehris** [comment]
Availability: Monday-Friday 5-9pm
ETA: 03/03/2023 EOD
